media-sound/musescore/musescore-3.6.2.recipe

```diff
@@ -71,7 +71,7 @@ BUILD()
 	make PREFIX=$appsDir/MuseScore UPDATE_CACHE=FALSE BUILD_PULSEAUDIO=OFF \
 		BUILD_JACK=OFF BUILD_PORTMIDI=OFF BUILD_PORTAUDIO=OFF BUILD_WEBENGINE=OFF \
 		USE_SYSTEM_FREETYPE=ON DOWNLOAD_SOUNDFONT=OFF BUILD_ALSA=OFF \
-		MUSESCORE_BUILD_CONFIG=release revision release
+		NO_RPATH=TRUE MUSESCORE_BUILD_CONFIG=release revision release
```

Comment:
Would be nice if you would add the separate option on a new line, makes it easier to follow :)
eg; 
```diff
- 		NO_RPATH=TRUE MUSESCORE_BUILD_CONFIG=release revision release
+ 		NO_RPATH=TRUE \
+		MUSESCORE_BUILD_CONFIG=release revision release
```
Also for the previous lines